sdcm/provision/azure/provisioner.py

```diff
@@ -37,10 +37,11 @@
 class AzureProvisioner(Provisioner):
     """Provides api for VM provisioning in Azure cloud, tuned for Scylla QA. """
 
-    def __init__(self, test_id: str, region: str, availability_zone: str,
+    def __init__(self, test_id: str, region: str, availability_zone: str, enable_azure_kms: bool = False,
```

Comment:
Remove the `enable_azure_kms` and start catching the `params` object.
Then use it to get the value for the `self._enable_azure_kms` attr.